draft-ietf-rtcweb-jsep.xml

```diff
@@ -461,7 +461,11 @@ setRemote(ANSWER) |                                   |
 
           <t>Upon receipt of trickled candidates, the receiving application
           will supply them to its ICE Agent. This triggers the ICE Agent to
-          start using the new remote candidates for connectivity checks.</t>
+          start using the new remote candidates for connectivity checks.
+          If BUNDLE is in use, each candidate is only supplied once, and
```

Comment:
Probably need xref here for BUNDLE.

Suggest making new paragraph here that can be specific to BUNDLE - might be a better fit for the ICE candidate section, which is where the m= section association is introduced.

Suggestion for text clarification: "When BUNDLE is in use, candidates are only exchanged for and associated with the m= section referenced by the sender's BUNDLE-tag, as described in <blah>".